web-platform/website/src/components/pages/account-settings/AccountSettingsNavigationBarWrapper.tsx

```diff
@@ -0,0 +1,11 @@
+import { AccountSettingsMobileNavigationBar } from "./AccountSettingsMobileNavigationBar";
```

Comment:
This seems like an overkill for a class esp we are using it just once. Should we kill it and inline calls from the caller?

Reply:
True lol